include/CL/opencl.hpp

```diff
@@ -6426,7 +6426,7 @@ class Program : public detail::Wrapper<cl_program>
 
         return detail::errHandler(
             pfn_clCreateProgramWithILKHR(
-                context(), static_cast<const void*>(IL.data()), IL.size(), &error);
+                context(), static_cast<const void*>(IL.data()), IL.size(), &error));
```

Comment:
I think these lines should be

```c++
        object_ = pfn_clCreateProgramWithILKHR(
                 context(), static_cast<const void*>(IL.data()), IL.size(), &error);
```

Reply:
Yes, it is incorrect to handle errors in this branch. I will modify the PR.